### DISCUSSION TITLE
GoPro Cameras in Twisters

### DISCUSSION THREAD

**Discussion by tetrahydra** [gopro/labs#1020] [General]
I tried to look for scenes where GoPro cameras were used but could not spot any scene. Does anyone have information?

**Comment by dnewman-gpsw** [comment]
Nothing more than the GoPro promo https://www.youtube.com/watch?v=pl8kv3QA7Pw

I wouldn't expect the scenes to be obvious, the GoPro shots in most features would be seamlessly integrated.  The only two exceptions are The Martian, where the GoPros became character or sorts (and the images where crap-ified), and Hardcore Henry (99.9% GoPro) where they kept the native lens distortion to improve the immersion.
